doc/en/array.txt

```diff
@@ -381,6 +450,50 @@
    $TEMPLATE$
       Function
    $NAME$
+      hb_AScan()
+   $CATEGORY$
+      API
+   $SUBCATEGORY$
+      Array
+   $ONELINER$
+      Scan array elements for a specified condition
+   $SYNTAX$
+      hb_AScan( <aArray>, <xSearch>, [<nStart>], [<nCount>], [<lExact>] ) --> <nPosition>
+   $ARGUMENTS$
+      <aArray>    Array to be scanned.
+      <xSearch>   Expression to search for in <aTarget>
+      <nStart>    Beginning subscript position at which to start the search. Default: 1
+      <nCount>    Number of elements to be scanned within <aArray>. Default: All elements.
+      <lExact>    Boolean flag specifying if an "Exact" search will be performed or not. Default: .F.
+   $RETURNS$
+      <nPosition> A numeric value of subscript position where <xSearch> was found, or 0 if nothing found.
+   $DESCRIPTION$
+      The function scans (left to right) for <xSearch> into <aArray> and returns <nPosition> of <aArray> in which <xSearch>
+      was found or 0 (zero) if nothing found. 
+      If <lExact> is .T., then an exact search will be performed.
+      When <xSearch> is a code block, the operation of the function is slightly different. See Ascan() for details.
+   $EXAMPLES$
+      proc main
+         local a := { "there", "here" }
+         ? AScan( a, "he" ) // --> 2
+         ? hb_AScan( a, "he",,,.F.) // --> 2
+         ? hb_AScan( a, "he",,,.T.) // --> 0 
+         wait
```

Comment:
Ok. Thanks for the hints.  I'll try to reformat it. 
(too late here.. I have to take a sleep, I think ;-))